scripts/lint-style.py

```diff
@@ -127,6 +130,13 @@ def small_alpha_vrachy_check(lines, path):
             errors += [(ERR_SAV, line_nr, path)]
     return errors
 
+def instance_check(lines, path):
+    errors = []
+    for line_nr, line in skip_string(skip_comments(enumerate(lines, 1))):
```

Comment:
It looks like some of the `skip_comments` stuff is not working

Reply:
`skip_comments` only skips `/- foobar -/`. Not `-- foobar`.

Reply:
That's surprising! I'd at least expect some explanation in `skip_comments` as to why, but that's a discussion for another PR.